src/radical/pilot/pmgr/launching/psi_j.py

```diff
@@ -0,0 +1,186 @@
+
+__copyright__ = "Copyright 2013-2016, http://radical.rutgers.edu"
+__license__   = "MIT"
```

Comment:
really off topic, but better to keep up to date
```python
__copyright__ = 'Copyright 2022, The RADICAL-Cybertools Team'
__license__   = 'MIT'
```

Reply:
done